Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.csproj

```diff
@@ -19,23 +19,23 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.Spanner.Data" Version="4.2.0"/>
```

Comment:
Would you mind removing these changes. As far as I can tell, these are just whitespace differences.